common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
We've been standardizing on `${msg | n, decode.utf8}`.  Also, is the html input from a user, or is it constructed in code?  If it is coming from a user, we should probably use `bleach`.

I'm not sure if using `bleach` with the defaults will be fine for all previous input.
@nedbat: Do you agree that we should start using bleach in places like this?  Should we update the docs as well?  Note that we already use bleach in some places, but I'm not sure if it is:
1. only with student input, or
2. arbitrarily used